ArduPlane/RC_Channel.cpp

```diff
@@ -62,6 +63,21 @@ void RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const aux_swi
     case INVERTED:
         plane.inverted_flight = (ch_flag == HIGH);
         break;
+
```

Comment:
This block should be in RC_Channel/RC_Channel.cpp (and the one in Copter removed)

Reply:
done